docs/tutorials/developer-journey/level-5/5.3-DEX-tutorial.md

```diff
@@ -0,0 +1,1376 @@
+# 5.3 Creating a decentralized exchange (DEX)
```

Comment:
I don't think it is a good idea to promote a tutorial using a deprecated standard. We'd either need to update this tutorial to use ICRC-1/2 or use this new simpler example https://github.com/dfinity/examples/tree/master/motoko/icrc2-swap

Reply:
I have rewritten this tutorial to use the ICRC-2 swap example.